src/processors/downgrade-unmodified-lines.js

```diff
@@ -0,0 +1,15 @@
+const fs = require( 'fs' );
+const downgradeUnmodifiedLines = require( '../lib/downgrade-unmodified-lines' );
+const differ = require( '../lib/differ' );
+const gitDiffCalculator = require( '../lib/git-diff-calculator' );
+
+const config = JSON.parse( fs.readFileSync( '.eslines.json', 'utf-8' ) );
+
+module.exports = function( report ) {
+	const remote = config.processors[ 'downgrade-unmodified-lines' ].remote || 'origin/master';
```

Comment:
Yes, I agree that being compatible with ESLint formatters is good. But adding an extra config argument to the formatter would still keep it compatible, although a bit more fragile: ESlint always passes only one argument and the second defaults to `undefined`. Just saying :)

I'm happy that this is finally merged and I'll move on to finishing the prettier processor now.